extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -1186,6 +1186,9 @@ export class AjaxMonitor extends BaseTelemetryPlugin implements IDependenciesPlu
 
                 if (input instanceof Request) {
                     ajaxData.requestUrl = input ? input.url : "";
+                } else if (input === "" ) {
+                    const location = getLocation();
```

Comment:
this seemed to be the way other code was accessing `window.location`

Reply:
well, if i wrote the code, I wouldn't have used `""`, but that's how NextJS 13's app router handles server-actions - I brought it up in [a discussions there](https://github.com/vercel/next.js/discussions/55405#discussioncomment-7006185), but after digging further into the code that appeared to be causing the issue (<https://github.com/vercel/next.js/blob/82eb6e6f4955dc90752a4875ccf86b7ac0c8d03a/packages/next/src/client/components/router-reducer/reducers/server-action-reducer.ts#L58>), it didn't seem that what the Next code was doing was actually contrary to how `fetch` is documented (or actually) works, so I figured the fix was better on this side in case there were other apps/libraries taking advantage of that behavior for `""` with `fetch`.